data/journal_check/bug_refs.json

```diff
@@ -0,0 +1,164 @@
+{
+    "bsc_1022527_FEATURE": {
+        "description": ".*wickedd.*ni_process_reap.*blocking waitpid.*",
+        "products": {
+            "sle-micro": ["5.0", "5.1"],
+            "opensuse": ["15.2", "15.3", "Tumbleweed"],
+            "microos": "Tumbleweed"
+          }
+    },
+    "bsc_1126272": {
+        "description": "Failed unmounting \/\\S+\\.|-- Reboot --|pam_systemd.*Failed to release session",
+        "products": {
+            "sle-micro": ["5.0", "5.1"],
+            "microos": "Tumbleweed"
+          }
+    },
+    "bsc_1022525_FEATURE": {
```

Comment:
This shouldn't actually be ignored. I don't know why the bug report was closed...

Reply:
Still happening in [SLE Micro](https://openqa.suse.de/tests/6251886#step/journal_check/23), so please keep it in the list. I will comment on the bug.